models/projects.go

```diff
@@ -8,32 +8,32 @@ import "gorm.io/gorm"
 type Project struct {
 	gorm.Model
 
-	Name          string
-	Desc          string `gorm:"size:2550"`
-	Tags          string
-	RepoLink      string
-	ComChannel    string
-	README        string
-	ProjectStatus bool `gorm:":default:false"`
+	Name          string `gorm:"column:name"`
+	Description   string `gorm:"size:2550;column:description"`
+	Tags          string `gorm:"column:tags"`
+	RepoLink      string `gorm:"column:repo_link"`
+	ComChannel    string `gorm:"column:com_channel"`
+	ReadmeLink    string `gorm:"column:readme_link"`
+	ProjectStatus bool   `gorm:"default:false;column:project_status"`
 
 	// for stats
-	LastPullTime int64
+	LastPullTime int64 `gorm:"column:last_pull_time"`
 
 	// stats table
-	CommitCount  uint
-	PRCount      uint
-	AddedLines   uint
-	RemovedLines uint
+	CommitCount  uint `gorm:"column:commit_count"`
+	PullCount    uint `gorm:"column:pull_count"`
+	AddedLines   uint `gorm:"column:added_lines"`
+	RemovedLines uint `gorm:"column:removed_lines"`
 
 	// list of students who contributed to the project (a string of usernames separated by comma(,))
-	Contributors string
+	Contributors string `gorm:"column:contributors"`
 
 	// list of URLs to PRs contributed to the project (a string of links separated by comma(,))
-	Pulls string
+	Pulls string `gorm:"column:pulls"`
 
 	// foreign keys
-	Mentor_id          int32
-	Mentor             Mentor `gorm:"ForeignKey:Mentor_id"`
-	SecondaryMentor_id int32
-	SecondaryMentor    Mentor `gorm:"ForeignKey:SecondaryMentor_id"`
+	Mentor_id          int32  `gorm:"column:mentor_id"`
+	Mentor             Mentor `gorm:"ForeignKey:Mentor_id;column:mentor"`
```

Comment:
I don't think `mentor` is a column, just the `mentor_id` is an actual column here. Mentor is preloaded by gorm.